x/genesis/client/cli/tx.go

```diff
@@ -1,16 +1,21 @@
 package cli
 
 import (
+	_ "encoding/json"
 	"fmt"
 	"github.com/cosmos/cosmos-sdk/client/flags"
 	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
 	tmjson "github.com/tendermint/tendermint/libs/json"
 	tmtypes "github.com/tendermint/tendermint/types"
-
-	"github.com/spf13/cobra"
+	"io/ioutil"
+	_ "io/ioutil"
```

Comment:
can we perhaps run goimports for the whole source code?

Reply:
I did some experimentation on this file, hence those unused import here.
I run it on this file, I think it should be ok for the other sources